packages/js-sdk/src/utils/gasMetering/gasMetering.js

```diff
@@ -0,0 +1,118 @@
+const { BN } = require("@openzeppelin/test-helpers");
```

Comment:
Is the idea that this is only to be used in development, never in production? If so we should add this to docs

Reply:
answered above